codebase2/codebase-sqlite/U/Codebase/Sqlite/Queries.hs

```diff
@@ -1441,14 +1454,12 @@ removeHashObjectsByHashingVersion hashVersion =
       WHERE hash_version = ?
 |]
 
--- | Drop and recreate the name lookup tables. Use this when resetting names to a new branch.
-resetNameLookupTables :: Transaction ()
-resetNameLookupTables = do
-  execute_ "DROP TABLE IF EXISTS term_name_lookup"
-  execute_ "DROP TABLE IF EXISTS type_name_lookup"
+-- | Ensure the name lookup tables exist.
+ensureNameLookupTables :: Transaction ()
```

Comment:
We can't just drop the tables every time anymore, but we still need to ensure they exist on enlil before using them.

If the new method of updating the index is fast enough we might consider using the index in UCM too; but that'd take a bit more looking into to see if it's even worth it.